src/task_run_diagnostic_reports.py

```diff
@@ -24,14 +26,41 @@ def auth(config):
     return ow
 
 
-def run_reports(config, reports):
-    orderly_web = auth(config)
+def run_reports(orderly_web, config, reports):
```

Comment:
Can you remind me why we're queueing up multiple reports here? It seems that we then have potentially 3 "queues" (or queue-ish things) - this task queue, the set of things sent here and the queue that orderly.server keeps. We also might move to using a "workflow" once those are fully supported through the runner

Reply:
This is just in case there are multiple diagnostic reports which should be spawned for a particular group/disease. Not sure how likely that is but I thought it was plausible that there could be some special cases..